config.example/containers/dgxie/dnsmasq.conf

```diff
@@ -34,3 +34,10 @@ dhcp-option=3,#GATEWAY#
 #dhcp-boot=tag:bios,"pxelinux.0"
 
 conf-dir=/etc/dnsmasq.d,*.conf
+
+# If the dhcp-ignore flag is specified in this fashion, only hosts configured with dhcp-host will be given DHCP
+dhcp-ignore=tag:!known 
```

Comment:
With this config in place, do you need to list the MACs in two places? Here and in `machines.json`?

Reply:
This is what assigns static dhcp leases, machines.json controls what 'profile' the pxe server assigns to the machine.
I don't think we should default to having this un-commented, because nothing will boot by default. Either way we need documentation on this.

Reply:
The logic behind leaving this here was so that DGXie would only start giving addresses out if it had been configured correctly. This was to prevent someone accidentally standing up multiple DHCP servers in their environment. Something worth documenting.